src/com/mindscriptact/mvcExpressLogger/minimalComps/components/Mvce_Label.as

```diff
@@ -39,8 +39,6 @@ public class Mvce_Label extends Mvce_Component {
 	protected var _text:String = "";
 	protected var _tf:TextField;
 
-	public var textFieldAutoSize:String = TextFieldAutoSize.LEFT;
```

Comment:
I had a need to be able to change label autosize... don't remember where.
 Maybe add a function instead public property? setAutoSize(textFieldAutoSize:String)